java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/numeric/ArrowFlightJdbcBaseIntVectorAccessorUnitTest.java

```diff
@@ -125,124 +126,104 @@ public static void setup() {
 
   @AfterClass
   public static void tearDown() throws Exception {
-    AutoCloseables.close(
-        bigIntVector, intVector, smallIntVector, tinyIntVector, int4Vector, int8Vector,
-        intVectorWithNull, rule);
+    AutoCloseables.close(bigIntVector, intVector, smallIntVector, tinyIntVector, int4Vector,
+        int8Vector, intVectorWithNull, rule);
   }
 
   @Test
   public void testShouldGetStringFromUnsignedValue() throws Exception {
     accessorIterator.assertAccessorGetter(int8Vector,
-        ArrowFlightJdbcBaseIntVectorAccessor::getString,
-        equalTo("18446744073709551615"));
+        ArrowFlightJdbcBaseIntVectorAccessor::getString, equalTo("18446744073709551615"));
   }
 
-  @Test
+  @Test(expected = SQLException.class)
   public void testShouldGetBytesFromIntVector() throws Exception {
     byte[] value = new byte[] {(byte) 0xaa, (byte) 0xbb, (byte) 0xcc, (byte) 0xdd};
 
-    accessorIterator
-        .assertAccessorGetter(intVector, ArrowFlightJdbcBaseIntVectorAccessor::getBytes,
-            CoreMatchers.is(value));
+    accessorIterator.assertAccessorGetter(intVector, ArrowFlightJdbcBaseIntVectorAccessor::getBytes,
+        CoreMatchers.is(value));
```

Comment:
Change this to use the new assert method that throws an exception

Reply:
fixed